count_ignores.sh

```diff
@@ -0,0 +1,40 @@
+current_dir=$(pwd)
+
+cd exercises
```

Comment:
because this always does `cd exercises`, the caller of this script must be at the root of https://github.com/exercism/rust when running it, which is more disadvantageous compared to being able to run it anywhere